lib/debloater/index.rb

```diff
@@ -56,21 +56,31 @@ def size
 
 
     def debloat!
+      _log "Old index definition:"
+      _log @sql
+
       [
-        "DROP INDEX IF EXISTS idx_debloat_new",
+        'DROP INDEX IF EXISTS idx_debloat_new',
         @sql.
           sub(/CONCURRENTLY/i, '').
-          sub(/CREATE (UNIQUE)? INDEX/i, 'CREATE \1 INDEX CONCURRENTLY').
+          sub(/CREATE\s+(UNIQUE\s+)?INDEX/i, 'CREATE \1INDEX CONCURRENTLY').
           sub(@name, 'idx_debloat_new'),
-        'BEGIN',
-        "ALTER INDEX #{@name} RENAME TO idx_debloat_old",
-        "ALTER INDEX idx_debloat_new RENAME TO #{@name}",
-        'DROP INDEX idx_debloat_old',
-        'COMMIT',
+        %{
+          BEGIN;
+          ALTER INDEX #{@name} RENAME TO idx_debloat_old;
+          ALTER INDEX idx_debloat_new RENAME TO #{@name};
+          DROP INDEX idx_debloat_old;
+          COMMIT;
+        },
```

Comment:
This is a single string now, instead of an Array.
It should still work, just checking if it was meant to be `%W{}` instead.